src/middleware/devtools.ts

```diff
@@ -1,3 +1,5 @@
+import '@redux-devtools/extension'
```

Comment:
Perhaps you'd know this better just make sure this doesn't add any unused runtime code.

Reply:
yeah, the final app code should be fine, but we should actually remove in dist. I'll see how I can do it.

Reply:
e9cf15e this should do.